src/AstPragma.h

```diff
@@ -17,17 +17,13 @@
 #pragma once
 
 #include "AstNode.h"
-#include "AstTransformer.h"
 #include <cassert>
 #include <ostream>
 #include <string>
 #include <utility>
-#include <vector>
 
 namespace souffle {
 
-class AstTranslationUnit;
-
 /**
  * @class AstPragma
  * @brief Representation of a global option
```

Comment:
Are we sure we need the default constructor?

Reply:
It's only used by the clone method, so I refactored it out.